EnglishTranslation-master/SKILL.tsv

```diff
@@ -4597,15 +4597,15 @@ SKILL_20150714_004596	Increases the damage dealt on an enemy with [Concentrated
 SKILL_20150714_004597	Increases the damage dealt on an enemy with [Caracole] by 1% per attribute level.
 SKILL_20150714_004598	Increases the damage dealt on an enemy with [Limacon] by 1% per attribute level.
 SKILL_20150714_004599	Increases the damage dealt on an enemy with [Retreat Shot] by 1% per attribute level.
-SKILL_20150714_004600	Gun Mastery: Accuracy
+SKILL_20150714_004600	Pistol Mastery: Accuracy
 SKILL_20150714_004601	Increases your accuracy by 30 per attribute level when equipped with a Pistol-type weapon.
 SKILL_20150714_004602	Increases the damage dealt on an enemy with [Cure] by 1% per attribute level.
 SKILL_20150714_004603	Increases the damage dealt on an enemy with [Heal] by 1% per attribute level.
-SKILL_20150714_004604	Enemies hit with [Strike] has 2% chance per Attribute level to fall under Stun.
-SKILL_20150714_004605	When using [Heal], you will receive the same effects too at a 2% chance per attribute level.
+SKILL_20150714_004604	Enemies hit with a [Strike] type attack have a 2% chance per Attribute level to fall under Stun.
+SKILL_20150714_004605	When using [Heal], adds the chance to automatically be healed at a 2% chance per attribute level.
```

Comment:
`Adds the chance to automatically be healed at a 2% chance per attribute level when using [Heal]`?